src/commands/src/shared/next-gen-parse.ts

```diff
@@ -3,7 +3,10 @@ import { Job } from '../../../entities/job';
 import { getDirectory } from '../../../job/jobHandler';
 import { CliCommandResponse, ExecuteCommandError, executeCliCommand } from '../helpers';
 
-const RSTSPEC_FLAG = '--rstspec=https://raw.githubusercontent.com/mongodb/snooty-parser/latest/snooty/rstspec.toml';
+const SNOOTY_PARSER_VERSION = process.env['SNOOTY_PARSER_VERSION']
+  ? `v${process.env['SNOOTY_PARSER_VERSION']}`
+  : 'main';
+const RSTSPEC_FLAG = `--rstspec=https://raw.githubusercontent.com/mongodb/snooty-parser/${SNOOTY_PARSER_VERSION}/snooty/rstspec.toml`;
```

Comment:
I forget, does this need a pre-pended `v` for the version tag?

Reply:
see comment below! removed the flag